src/tutorials/ping.rs

```diff
@@ -136,32 +136,37 @@
 //! [plaintext](crate::plaintext). It only cares about _what_ messages are sent
 //! on the network.
 //!
+//!
 //! The two traits [`Transport`] and [`NetworkBehaviour`] allow us to cleanly
 //! separate _how_ to send bytes from _what_ bytes to send.
 //!
 //! With the above in mind, let's extend our example, creating a [`ping::Behaviour`](crate::ping::Behaviour) at the end:
```

Comment:
With the change below, "creating" is no longer accurate, i.e. the code below is not actually instantiating a `Behaviour`. Can you either update the comment above, or actually instantiate the `Behaviour` in the code below?

``` rust
let behaviour = Behaviour::default();

Ok(())
```